src/architecture/msgPayloadDefCpp/OpNavCOBMsgPayload.h

```diff
@@ -29,11 +29,11 @@ typedef struct
 OpNavCOBMsgPayload
 //@endcond
 {
-    uint64_t timeTag;             //!< --[ns]   Current vehicle time-tag associated with measurements
```

Comment:
:small_red_triangle_down: This commit also can be squashed into the commit where the message typedef is first added.